public/app/features/alerting/unified/components/rule-editor/alert-rule-form/AlertRuleForm.tsx

```diff
@@ -109,19 +98,10 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
       return formValuesFromPrefill(prefill);
     }
 
-    if (queryParams.has('defaults')) {
```

Comment:
`AlertRuleForm` is no longer responsible for parsing data from query string. Instead, other components will pass the default as `prefill`